roles/sriov-dp-install/charts/sriov-net-dp/templates/sriovdp-daemonset.yaml

```diff
@@ -14,38 +14,41 @@ spec:
         tier: node
         app: sriovdp
     spec:
+      hostNetwork: true
+      hostPID: true
       nodeSelector:
         beta.kubernetes.io/arch: amd64
-      serviceAccountName: {{ .Release.Name }}-sriov-device-plugin
       tolerations:
-      - effect: NoSchedule
-        key: cmk
-        operator: Equal
-        value: "true"
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: {{ .Release.Name }}-sriov-device-plugin
       containers:
       - name: kube-sriovdp
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
Forcing latest image tag will remove the possibility to use other specific versions